integration/apps/rack/app/datadog.rb

```diff
@@ -3,12 +3,10 @@
 
 Datadog.configure do |c|
   c.diagnostics.debug = true if Datadog::DemoEnv.feature?('debug')
-  c.analytics_enabled = true if Datadog::DemoEnv.feature?('analytics')
+  c.analytics.enabled = true if Datadog::DemoEnv.feature?('analytics')
   c.runtime_metrics.enabled = true if Datadog::DemoEnv.feature?('runtime_metrics')
 
-  if Datadog::DemoEnv.feature?('tracing')
-    c.use :rack, service_name: 'acme-rack'
-  end
+  c.use :rack, service_name: 'acme-rack' if Datadog::DemoEnv.feature?('tracing')
```

Comment:
:rubocop:

Reply:
Yeah... To be honest I think our rubocop config is quite picky. Sometimes I do a one-character change, then push, and boom it broke because it's now over/under the size threshold where rubocop decides THE CODE MUST BE REFORMATTED NOW.